.gitignore

```
@@ -84,3 +84,14 @@ yarn-debug.log*
 
 # ignore Brewfile changes
 Brewfile.lock.json
+
+# Ignore PostgreSQL upgrade files
+pg_upgrade_dump_*.custom
+pg_upgrade_dump_*.log
+pg_upgrade_dump_globals.sql
+pg_upgrade_internal.log
+pg_upgrade_server.log
+pg_upgrade_utility.log
```

Is it worth having this?

```suggestion
pg_upgrade_*
```

---

Thanks Joe - I will remove it. I wasn't sure what to do with it.